development/templates/development/my_bots.html

```diff
@@ -33,6 +37,15 @@ <h1>My Bots</h1>
     <div class="col-12">
         <a class="btn btn-primary" href="{% url 'development:addbot' %}">Add new</a>
     </div>
+    <script>
```

Comment:
??????

Reply:
Yes, It`s true. 

I will move to js file